### ISSUE TITLE
Venv acclerate code error

### ISSUE THREAD

**Issue by Jaeunkobe** [bmaltais/kohya_ss#2300]
Hello, I am a designer who wants to teach LORA with your program.
I am currently installing and using kohya_ss using a stablity matrix, but this error code appears in the last stage of learning.

Since I'm not a developer, I used the stability matrix, but the error code appears in an alien language and I can't read it. I need help

> 19:32:30-096877 INFO     Start training LoRA Standard ...                      
> 19:32:30-096877 INFO     Validating model file or folder path                  
>                          stabilityai/stable-diffusion-xl-base-1.0 existence... 
> 19:32:30-097874 INFO     ...valid                                              
> 19:32:30-098870 INFO     Validating output_dir path C:/Users/WY/Desktop/Lora   
>                          Training/ControlAltAi Lora\model existence...         
> 19:32:30-098870 INFO     ...valid                                              
> 19:32:30-099867 INFO     Validating train_data_dir path                        
>                          C:/Users/WY/Desktop/Lora Training/ControlAltAi        
>                          Lora\img existence...                                 
> 19:32:30-100863 INFO     ...valid                                              
> 19:32:30-100863 INFO     reg_data_dir not specified, skipping validation       
> 19:32:30-100863 INFO     Validating logging_dir path C:/Users/WY/Desktop/Lora  
>                          Training/ControlAltAi Lora\log existence...           
> 19:32:30-101860 INFO     ...valid                                              
> 19:32:30-101860 INFO     log_tracker_config not specified, skipping validation 
> 19:32:30-102857 INFO     resume not specified, skipping validation             
> 19:32:30-102857 INFO     vae not specified, skipping validation                
> 19:32:30-102857 INFO     lora_network_weights not specified, skipping          
>                          validation                                            
> 19:32:30-103853 INFO     dataset_config not specified, skipping validation     
> 19:32:30-103853 INFO     Folder 40_Baemin Moolsung: 70 images found            
> 19:32:30-104850 INFO     Folder 40_Baemin Moolsung: 2800 steps                 
> 19:32:30-104850 INFO     Total steps: 2800                                     
> 19:32:30-105846 INFO     Train batch size: 1                                   
> 19:32:30-105846 INFO     Gradient accumulation steps: 1                        
> 19:32:30-106843 INFO     Epoch: 1                                              
> 19:32:30-106843 INFO     Regulatization factor: 1                              
> 19:32:30-106843 INFO     max_train_steps (2800 / 1 / 1 * 1 * 1) = 2800         
> 19:32:30-107840 INFO     stop_text_encoder_training = 0                        
> 19:32:30-107840 INFO     lr_warmup_steps = 280                                 
> 19:32:30-108837 INFO     Saving training config to C:/Users/WY/Desktop/Lora    
>                          Training/ControlAltAi Lora\model\ControlAltAi         
>                          Lora_20240416-193230.json...                          
> 19:32:30-109833 INFO     .\venv\scripts\accelerate launch                      
>                          --mixed_precision="bf16" --num_processes=1            
>                          --num_machines=1 --num_cpu_threads_per_process=32     
>                          "C:\Users\WY\Downloads\StabilityMatrix-win-x64\Data\Pa
>                          ckages\kohya_ss/sd-scripts/sdxl_train_network.py"     
>                          --bucket_no_upscale --bucket_reso_steps=64            
>                          --cache_latents --enable_bucket --min_bucket_reso=256 
>                          --max_bucket_reso=2048 --huber_c="0.1"                
>                          --huber_schedule="snr" --learning_rate="4e-08"        
>                          --logging_dir="C:/Users/WY/Desktop/Lora               
>                          Training/ControlAltAi Lora\log" --loss_type="l2"      
>                          --lr_scheduler="cosine" --lr_scheduler_num_cycles="1" 
>                          --lr_warmup_steps="280"                               
>                          --max_data_loader_n_workers="0" --max_grad_norm="1"   
>                          --resolution="1024,1024" --max_train_steps="2800"     
>                          --mem_eff_attn --min_timestep=0                       
>                          --mixed_precision="bf16" --network_alpha="1"          
>                          --network_dim=8 --network_module=networks.lora        
>                          --no_half_vae --optimizer_type="AdamW8bit"            
>                          --output_dir="C:/Users/WY/Desktop/Lora                
>                          Training/ControlAltAi Lora\model"                     
>                          --output_name="ControlAltAi Lora"                     
>                          --pretrained_model_name_or_path="stabilityai/stable-di
>                          ffusion-xl-base-1.0" --save_every_n_epochs="1"        
>                          --save_model_as=safetensors --save_precision="bf16"   
>                          --text_encoder_lr=4e-08 --train_batch_size="1"        
>                          --training_comment="Trigger with Moolsung"            
>                          --train_data_dir="C:/Users/WY/Desktop/Lora            
>                          Training/ControlAltAi Lora\img" --unet_lr=0.0001      
>                          --xformers                                            
> '.\venv\scripts\accelerate'��(��) ���� �Ǵ� �ܺ� ����, ������ �� �ִ� ���α׷�, �Ǵ�
> ��ġ ������ �ƴմϴ�.
>

**Comment by bmaltais** [comment]
Helli @Jaeunkobe I have been fighting code update ever since I had to change how the GUI work due to a security risk report on how external commands were run. This introduced a whole slew of issues... I have been re-writing a lot of the code to work around this and I has caused a lot of issues for users. I think I am getting closer to something that work well and securelly.

For now, you should use the `dev` branch code as it is the one that has the most chance to run properly... still fighting bugs discovered by user... but I have faith it is improving ;-)

So here is how to do it:

```
git fetch origin
git checkout dev
git pull
```

Then run gui.bat or gui.sh from there.

Hope this help.
